### PR TITLE
Ipxe changes

### PR DESCRIPTION
**Pull request by jcpowermac**
Used `packer: true` because my home lab does not have the same subnet as Austin lab.  Will add further testing.

```
➜  ansible git:(06dfcc1) ✗ ansible-playbook -e '{"packer": true}' site_discovery.yml

PLAY [Configure deployment node for site discovery] ***************************

GATHERING FACTS ***************************************************************
ok: [localhost]

TASK: [dhcp-server | install dhcp server package] *****************************
ok: [localhost]

TASK: [dhcp-server | Configure dhcpd.conf] ************************************
ok: [localhost]

TASK: [dhcp-server | Configure ipxe.conf] *************************************
ok: [localhost]

TASK: [dhcp-server | Copy ipxe-option-space.conf] *****************************
ok: [localhost]

TASK: [dhcp-server | enable dhcpd service] ************************************
skipping: [localhost]

TASK: [Check for local copy of Hanlon Microkernel] ****************************
ok: [localhost]

TASK: [Download Hanlon Microkernel] *******************************************
changed: [localhost]

TASK: [Copy Hanlon Microkernel to /opt/hanlon/image] **************************
skipping: [localhost]

TASK: [Add a Microkernel Image to Hanlon] *************************************
skipping: [localhost]

TASK: [Add a Discover Only Model to Hanlon] ***********************************
skipping: [localhost]

TASK: [Add a Discover Only Policy to Hanlon] **********************************
skipping: [localhost]

PLAY RECAP ********************************************************************
localhost                  : ok=7    changed=1    unreachable=0    failed=0
```

### CODE REVIEW

**Comment by jcpowermac** [comment]
Results from virtual machine running DHCPd on the correct subnet.
image

File: ansible/roles/base/tasks/main.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@
     name: docker-py
     state: present
     extra_args: "--upgrade"
-    when: not offline
+  when: not offline
```
Comment (jdextercsc) [inline ansible/roles/base/tasks/main.yml]: This is duplicated in https://github.com/csc/kragle/pull/37/files

**Comment by jcpowermac** [comment]
Removed update to resolve the merge conflict with PR #37

**Comment by jdextercsc** [comment]
code review :+1:

**Comment by mtnbikenc** [comment]
Tested this branch locally.  Successfully ran a node discovery.  :+1:

**Pull request merged**